test/runtests.jl

```diff
@@ -179,24 +192,24 @@ edge_case_imgs = [
                 @test PNGFiles.save(fpath, image) == 0
             end
             @testset "read" begin
-                global read_in = PNGFiles.load(fpath)
-                @test read_in isa Matrix
+                global read_in_pngf = PNGFiles.load(fpath)
+                @test read_in_pngf isa Matrix
             end
             @testset "compare" begin
-                @test all(read_in .== func_in(image))
+                @test all(read_in_pngf .== func_in(image))
             end
             global read_in_immag = _standardize_grayness(ImageMagick.load(fpath))
             @testset "$(case): ImageMagick read type equality" begin
-                @test eltype(read_in) == eltype(read_in_immag)
+                @test eltype(read_in_pngf) == eltype(read_in_immag)
             end
             @testset "$(case): ImageMagick read values equality" begin
-                @test all(read_in .≈ read_in_immag)
+                @test all(read_in_pngf .≈ read_in_immag)
             end
             path, ext = splitext(fpath)
             newpath = path * "_new" * ext
-            PNGFiles.save(newpath, read_in)
+            PNGFiles.save(newpath, read_in_pngf)
             @testset "$(case): IO is idempotent" begin
-                @test all(read_in .≈ PNGFiles.load(newpath))
+                @test imdiff(read_in_pngf, PNGFiles.load(newpath)) < 0.01
```

Comment:
Perhaps you would want to try ReferenceTests, which is lighter than `Images`

Reply:
I did think about this, but it's just a test dependency. 
ReferenceTests looks interesting, I wasn't aware of that! thanks

Reply:
Thanks JohnnyChen94, I'll look into ReferenceTests.jl!